### ISSUE TITLE
modprobe zfs required after every boot

### ISSUE THREAD

**Issue by Germano0** [openzfs/zfs#10462]
Type | Version/Name
 --- | ---
Distribution Name	| CentOS
Distribution Version	| 8.2
Linux Kernel	| 4.18.0-193.6.3.el8_2.x86_64
Architecture	| x86_64
ZFS Version	| 0.8.4-1
SPL Version	| 0.8.4-1

### Describe the problem you're observing
Every time I boot my machine, I need to run `modprobe zfs` in order to load zfs module
### Describe how to reproduce the problem
```
# zfs mount -l zpool_1/stuff
The ZFS modules are not loaded.
Try running '/sbin/modprobe zfs' as root to load them.
# modprobe zfs
```

**Comment by behlendorf** [comment]
@Germano0 if `blkid` detects a real physical device with a ZFS pool on it the modules will be automatically loaded.  Additionally, if the `/usr/lib/udev/rules.d/90-zfs.rules` file is installed running any `zfs` or `zpool` command should result in the auto-loader pulling in the modules (a6cc975).  Can you check that it's installed. Alternately, you can always force the module to load on every boot by uncommenting the last line in `/etc/modules-load.d/zfs.conf`.

**Comment by Germano0** [comment]
I close this bugreport because with 0.8.4 zfs module loads correctly after system boot.
I had the problem because I experienced dkms build failure (https://github.com/openzfs/zfs/issues/10463) after an upgrade and after you run `dkms autoinstall` the zfs module is not loaded.
Since with past ZFS versions I had to manually load zfs module even if dkms was okay, yesterday I tought it was the same problem so I opened this ticket. But instead as I said, 0.8.4 works fine, you only have to manually load zfs module if you are running `dkms autoinstall` after a zfs update
